rasa/server.py

```diff
@@ -442,7 +443,7 @@ def create_app(
         except Exception as e:
             logger.debug(traceback.format_exc())
             raise ErrorResponse(
-                500, "ConversationError", f"An unexpected error occurred. Error: {e}",
+                500, "ConversationError", f"An unexpected error occurred. Error: {e}"
```

Comment:
can you please make sure you have the same black version as we install in travis (black==19.10b0)? This caused a lot of unnecessary diffs

Reply:
yea I don't know why that happened I installed it from the requirements file which I thought is what travis was using as well, I've updated my venv tho.